src/components/Game.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react'
+import Board from './Board'
+
+//component
+const Game = () => {
+  const initialBoard = ['', '', '', '', '', '', '', '', '']
+  const [gameHistory, setGameHistory] = useState<GameState[]>([initialBoard])
+  const [isXNext, setIsXNext] = useState<boolean>(true)
+
+  /**
+   * pointer function for handling click events from square component
+   * adds current board state to history[] state
+   * @param squares
+   */
+  const addGameHistory = (squares: string[], sliderValue: number) => {
+    setIsXNext(!isXNext)
```

Comment:
In order for this to work when moving backward in time you need to tie this to the lifecycle of the sliderValue instead